site/content/posts/2021-06-10-docs-wg.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Contour Technical Documentation Working Group Kick Off
+excerpt: Announcing Contour Technical Documentation Working Group
+author_name: Contour Team
+author_avatar: /img/contributors/orlin-vasilev.jpg
+categories: [kubernetes]
+tags: ['Contour Team', 'Abigail McCarthy', 'Docs' ,'Docs-WG', 'Jonas Rosland', 'Orlin Vasilev']
+date: 2021-06-10
+slug: docs-wg
+---
+
+
+
+# Join the Contour Tech Docs Working Group
```

Comment:
This should be the title on line 1, then this line can be removed

Reply:
fixed